components/user_list.jsx

```diff
@@ -32,9 +32,14 @@ export default class UserList extends React.PureComponent {
         rowComponentType: UserListRow,
     }
 
+    constructor(props) {
+        super(props);
+        this.container = React.createRef();
```

Comment:
can we rename this to match what we are doing in other PRs
```suggestion
        this.containerRef = React.createRef();
```

Reply:
okay @Willyfrog